packages/catalog-process/src/services/catalogService.ts

```diff
@@ -1347,6 +1350,42 @@ export function catalogServiceBuilder(
         newEservice
       );
 
+      await repository.createEvent(event);
+    },
+    async deleteRiskAnalysis(
+      eserviceId: EServiceId,
+      riskAnalysisId: RiskAnalysisId,
+      authData: AuthData
+    ): Promise<void> {
+      logger.info(
+        `Deleting Risk Analysis ${riskAnalysisId} for EService ${eserviceId}`
+      );
+      const eservice = await retrieveEService(eserviceId, readModelService);
+
+      assertRequesterAllowed(eservice.data.producerId, authData.organizationId);
```

Comment:
In the Scala implementation we don't have `assertRequesterAllowed` in this endpoint. Should we keep this check here?

Reply:
> In the Scala implementation we don't have `assertRequesterAllowed` in this endpoint. Should we keep this check here?

I think it would be better if we keep it

Reply:
good catch 🚀